File: beginner/c/1866.c
#include <stdio.h>
#include <string.h>

int main() {
    
    int i, j, n, x, cont = 1, v, soma;

    scanf("%d", &n);

    for(i = 0; i < n; i++){

        soma = 0;

        scanf("%d", &v);

        for(j = 1; j <= v; j++){
            if(j % 2 != 0){
                soma += cont;
            } else{
                soma -= cont;
            }
        }

        printf("%d\n", soma);
    }


    return 0;
}

File: beginner/java/1866.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);
        
        int i, j, n, cont = 1, v, soma;

        n = entrada.nextInt();

        for(i = 0; i < n; i++){

            soma = 0;

            v = entrada.nextInt();

            for(j = 1; j <= v; j++){
                if(j % 2 != 0){
                    soma += cont;
                } else{
                    soma -= cont;
                }
            }

            System.out.printf("%d%n", soma);
        }
    }
}
